tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature

```diff
@@ -434,3 +434,81 @@ Feature: upload file
     Then the HTTP status code should be "204"
     And for user "Brian" the content of the file "/textfile.txt" of the space "new-space" should be ""
     And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
+
+
+  Scenario Outline: user replaces a file with an empty file
```

Comment:
Can we demonstrate this bug report in scenario?
https://github.com/owncloud/ocis/issues/8003#issuecomment-1867226950

Reply:
No, through API replacing a file with an empty file works fine. The bug only seems to exist on the web.

Reply:
it must be a TUS upload